### ISSUE TITLE
Option to scrape additional channels from TVGuide

### ISSUE THREAD

**Issue by subby2006** [cabernetwork/provider_video_daddylive#23]
Is there a way to scrape additional channels from the tvguide plugin and include them in the xmltv? I've tried editing the channel_list.json and forced an EPG refresh but the additional channels don't appear.

**Comment by rocky4546** [comment]
Just checked H&I.  It does not appear in the 24/7 list on DL, so it does not exist and cannot be added...  It must exist on the 24/7 list to be added.

**Comment by subby2006** [comment]
Yeah these are some extra channels not on DL. I have an IPTV Merge with DL channels and a few extra ones  on my Kodi setup and was wondering if I could use the TV Guide scraper here on Cabernet to get epg data for the channels not on DL

**Comment by dubmn** [comment]
So I found an interesting glitch with the DL 24/7 list.  The "Bally Sports" family of channels do not show up on the Full List tab, but are there when you go to the B tab.  Therefore, I cannot get them into Cabernet.  I haven't looked at the scraper code yet, but the letter tabs are on the same html file as the Full List one.  Could you possibly just continue down the document on the scrape and kill the dupes that appear?
